docs/GIT_FLOW.md

```diff
@@ -0,0 +1,145 @@
+## Branching
+### Gitflow:
+
+1. The branch `develop` is created from `main`
+2. The branch `release` is created from `develop`
```

Comment:
what was the purpose of the release branch again?

Reply:
Creating an ephemeral ‘release’ branch would allow the team to decide on a specific snapshot of ‘develop’ to test and polish for release, while still allowing development on the ‘develop’ branch. 

When ‘release’ is ready, it gets merged into ‘main’ and ‘develop’, then deleted.